docs/versioned-plugins/inputs/elastic_serverless_forwarder-v0.1.0.asciidoc

```diff
@@ -0,0 +1,307 @@
+:plugin: elastic_serverless_forwarder
+:type: input
+:no_codec:
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: v0.1.0
+:release_date: 2022-11-18
+:changelog_url: https://github.com/logstash-plugins/logstash-input-elastic_serverless_forwarder/blob/v0.1.0/CHANGELOG.md
+:include_path: ../include/6.x
+///////////////////////////////////////////
+END - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+
+:esf-name: Elastic Serverless Forwarder
+
+[id="{version}-plugins-{type}s-{plugin}"]
+
+=== {esf-name} input plugin {version}
+
+include::{include_path}/plugin_header.asciidoc[]
+
+==== Description
+
+Using this input you can receive events from {esf-name} over http(s) connections to the configured <<{version}-plugins-{type}s-{plugin}-port>>.
+
+[id="{version}-plugins-{type}s-{plugin}-ext-field"]
+===== Minimum Configuration
+[cols="3a,2a"]
+|=======================================================================================================================
+|SSL Enabled              |SSL Disabled
+
+|
+
+[source,subs="+macros,attributes"]
+----
+input {
+  {plugin} {
+    <<{version}-plugins-{type}s-{plugin}-port,port>> => 8080
+    <<{version}-plugins-{type}s-{plugin}-ssl_certificate,ssl_certificate>> => "/path/to/logstash.crt"
+    <<{version}-plugins-{type}s-{plugin}-ssl_key,ssl_key>> => "/path/to/logstash.key"
```

Comment:
elastic_serverless_forwarder {
    port => 8080
    ssl_certificate => "/path/to/logstash.crt"
    ssl_key => "/path/to/logstash.key"